nautobot_ssot/integrations/infoblox/signals.py

```diff
@@ -84,14 +86,14 @@ def nautobot_database_ready_callback(sender, *, apps, **kwargs):  # pylint: disa
     )
     range_custom_field.content_types.add(ContentType.objects.get_for_model(Prefix))
 
-    mac_address_comment_custom_field, _ = CustomField.objects.get_or_create(
+    mac_address_custom_field, _ = CustomField.objects.get_or_create(
```

Comment:
We should probably have documentation around this being done so users aren't surprised when they suddenly see a new CustomField on their IPAddresses.